### DISCUSSION TITLE
Error when launching Ansys Prime Server

### DISCUSSION THREAD

**Discussion by blazvelkavrh** [ansys/pyprimemesh#566] [Q&A] [answered]
Hello!

I would like to prepare mesh on imported CAD geometry with PyPrimeMesh for further use in PyFluent. However, when trying to launch Ansys Prime Server:
```
from ansys.meshing import prime

prime_client = prime.launch_prime()
model = prime_client.model
```
I get an error for `prime.launch_prime()`: FileNotFoundError: No valid Ansys Prime Server found to launch.

Based on the error, I would assume I am missing something with installation of Ansys, however, searching anything in connection to Ansys Prime Server brings me to PyPrimeMesh in almost all cases.

I am using:
- Windows 11
- Ansys 2023 R1
- Python 3.10
- ansys-meshing-prime 0.4.0

On PyPrimeMesh documentation page ([link](https://prime.docs.pyansys.com/version/0.4/getting_started/index.html)), Fluids PrepPost (CFD) license is written as a requirement. I can use Ansys Fluent without any issues, and based on the Ansys Licensing Settings, it is using cfd_preppost license successfully.

I would kindly ask you for help/suggestions on how to make this work.

Thank you very much for the answers!

Best,
Blaz

**Comment by waltersma** [comment, marked as answer]
Hi Blaz,

Currently the client has one-to-one compatibility with a server release, documented in [getting started](https://prime.docs.pyansys.com/version/stable/getting_started/index.html).  You will need to upgrade to Ansys 2023 R2 to use 0.4.0.  Or you can use 0.2.0 or 0.3.0 for 2023R1 depending on whether you have the service pack installed or not.

Hope this helps!
